src/sdo/sood/data/sdo_ml_v2_dataset.py

```diff
@@ -216,7 +224,11 @@ def __init__(self,
                  start=None,
                  end=None,
                  freq=None,
-                 shuffle: bool = False):
+                 shuffle: bool = False,
+                 train_val_split_strategy: str = "temporal",
+                 train_val_split_ratio: float = 0.7,
+                 train_val_split_temporal_chunk_size: str = "14d"
```

Comment:
a default temporal chunk size of 14 days is assumed, this should be sufficient that structures are dissimilar between train and validation sets. Notably there is no temporal gap between the chunks